docker/docker-dev/docker-compose.yml

```diff
@@ -35,6 +35,9 @@ services:
      - "8080:80"
     environment:
       GRAPHITE_CLUSTER_SERVERS: metrictank:6060
+      WSGI_PROCESSES: 4
+      WSGI_THREADS: 25
+      GRAPHITE_STATSD_HOST: statsdaemon
```

Comment:
GRAPHITE_* are graphite settings, defined in graphite's local_settings.py
WSGI_* are wsgi settings, defined in the WSGIDaemonProcess directive in Apache.